src/Controller/NphOrderLookupController.php

```diff
@@ -29,6 +31,31 @@ public function index(
         SiteService $siteService,
         NphParticipantSummaryService $participantSummary
     ): Response {
+        $recentOrders = $this->em->getRepository(NphOrder::class)->getRecentOrdersBySite($siteService->getSiteId());
+        foreach ($recentOrders as &$order) {
+            $order->participant = $participantSummary->getParticipantById($order->getParticipantId());
+        }
+        $idForm = $this->getIdForm($request, $siteService, $participantSummary);
+        return $this->generateOrderLookupView('program/nph/order/orderlookup.html.twig', $idForm, $recentOrders);
+    }
+
+    /**
+     * @Route("/nph/biobank/orders", name="nph_order_lookup_biobank")
```

Comment:
I created a NphBiobankController to hold all the biobank routes. Could you move this to that controller? You might need to update your develop branch in order to view that controller.